src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -29,9 +29,10 @@ public static class KnownRoutes
         public const string StoreInstancesRoute = StudiesRouteSegment;
         public const string StoreInstancesInStudyRoute = StudiesRouteSegment + "/{" + KnownActionParameterNames.StudyInstanceUid + "}";
 
-        public const string CreateWorkitemInstancesRoute = WorkitemsRouteSegment;
+        public const string AddWorkitemInstancesRoute = WorkitemsRouteSegment;
         public const string RetrieveWorkitemInstancesRoute = WorkitemsRouteSegment + "/{" + KnownActionParameterNames.WorkItemInstanceUid + "}";
         public const string SearchWorkitemInstancesRoute = WorkitemsRouteSegment;
+        public const string CancelWorkitemInstancesRoute = WorkitemsRouteSegment + "/{" + KnownActionParameterNames.WorkItemInstanceUid + "}/cancelrequest";
```

Comment:
Should we create a constant for cancelrequest

Reply:
Done